util/pathutil.py

```diff
@@ -107,11 +107,11 @@ def test(r, space):
 
     return (namedtuple('PathInfo', 'space zone group subpath'.split())
```

Comment:
This change is not needed?

Reply:
Code looks fine, yes. This only fixes a flake8 error. Revert?

Reply:
Should not be an error, passes in development?

Reply:
Probably due to running older flake8/pyflakes versions. Reverted the commit and pushed a new one.